settings.py

```diff
@@ -115,4 +119,20 @@ def get_listen_ip(self):
     def get_listen_port(self):
         return self['listen'].split(':')[1]
 
+    def check_user(self, user, pswd):
+        try:
+            au = self['user']
+            pw = self['password']
+            logging.info("check_user(" + user + "," + pswd + "), looking for (" + au + "," + pw + ")")
```

Comment:
Agreed -- that was mostly for debugging. Do you want me to remove the ,pswd, or just remove them when it's integrated? An alternative is to log the md5 hash of the passwords, which IS an acceptable practice.